python/example_code/ses/ses_replicateidentities/ses_replicateidentities.py

```diff
@@ -0,0 +1,343 @@
+        if bounce_topic == '':
+            pass
+        else:
+            try:
+                ses_dest_client.\
+                               ses_identity_notif_topic(
+                                                        Identity=identity,
+                                                        NotifType='Bounce',
+                                                        SnsTopic=bounce_topic
+                                                        )
+            except ClientError:
+                print("Invalid ARN")
+
+        delivery_topic = input("Enter ARN of delivery topic: ")
+        if delivery_topic == '':
+            pass
+        else:
+            try:
+                ses_dest_client.\
+                               ses_identity_notif_topic(
+                                                        Identity=identity,
+                                                        NotifType='Delivery',
+                                                        SnsTopic=delivery_topic
+                                                        )
+            except ClientError:
+                print("Invalid ARN")
+
+        complaint_topic = input("Enter ARN of complaint topic: ")
+        if complaint_topic == '':
+            pass
+        else:
+            try:
+                ses_dest_client.\
+                               ses_identity_notif_topic(
+                                                        Identity=identity,
+                                                        NotifType='Complaint',
+                                                        SnsTopic=complaint_topic
+                                                        )
+            except ClientError:
+                print("Invalid ARN")
+    elif ask == 'no':
+        return
+
+
+# Main
+SRC_REGION = input("Which region do you want to replicate from? ")
+if SRC_REGION in regions:
+    ses_source_client = S.client('ses', region_name=SRC_REGION)
+    region_email_identities = []
+    region_dom_identities = []
+    # Added pagination for listing SES identities.
+    paginator = ses_source_client.get_paginator('list_identities')
+    response_iterator = paginator.paginate(PaginationConfig={
+                                                             'MaxItems': 20,
+                                                             'PageSize': 20
+                                                            }
+                                           )
+    for x in response_iterator:
+        for y in x['Identities']:
+            if '@' in y:
+                region_email_identities.append(y)
+            else:
+                region_dom_identities.append(y)
+    print ("Email addresses in source region:")
+    print(region_email_identities)
+    print ("Domains in source region:")
+    print(region_dom_identities)
+
+else:
+    sys.exit("Region entered invalid. \
+             Please enter a region where SES is available.")
```

Comment:
Instead of sys.exit, maybe re-print the list of valid regions and re-ask for user selection?

Reply:
Added while loop that runs until correct input is given.